### ISSUE TITLE
Intel: Linux: OpenGL: SSBU glitches in matches

### ISSUE THREAD

**Issue by A-w-x** [yuzu-emu/yuzu#7983]
basically this: https://gitlab.freedesktop.org/mesa/mesa/-/issues/5991

but its not a mesa bug because the game has no issues in other emulators with ogl

since (i know that it sounds weird) ogl is a lot faster than vulkan on intel, it would be really cool if this can be fixed

it only affects ogl
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/8193685/yuzu_log.txt)
l

**Comment by v1993** [comment]
I mean, different emulators work a bit differently. It very well can be a mesa bug, especially given that linked issue was opened by yuzu dev.

**Comment by A-w-x** [comment]
true, but i had that bug in all emulators before, even in windows
i did not test windows now, but the game is working with ogl somewhere, and it is working with anv now so it seems to be the case that the emulator is doing something not supported by the gpu, that does not mean its a bug in mesa but how it works (limitations?)

**Comment by benderscruffy** [comment]
Is this still an issue ???

**Comment by A-w-x** [comment]
no idea, sold the device

**Comment by benderscruffy** [comment]
Please close then
